kobo/settings/base.py

```diff
@@ -274,36 +274,48 @@
     ),
     'USER_METADATA_FIELDS': (
         json.dumps([
-            {'name': 'organization', 'required': False},
-            {'name': 'organization_website', 'required': False},
-            {'name': 'sector', 'required': False},
-            {'name': 'gender', 'required': False},
-            {'name': 'bio', 'required': False},
-            {'name': 'city', 'required': False},
-            {'name': 'country', 'required': False},
-            {'name': 'twitter', 'required': False},
-            {'name': 'linkedin', 'required': False},
-            {'name': 'instagram', 'required': False},
+            {'name': 'full_name', 'required': False,},
+            {'name': 'organization', 'required': False,},
+            {'name': 'organization_website', 'required': False,},
+            {'name': 'sector', 'required': False,},
+            {'name': 'gender', 'required': False,},
+            {'name': 'bio', 'required': False,},
+            {'name': 'city', 'required': False,},
+            {'name': 'country', 'required': False,},
+            {'name': 'twitter', 'required': False,},
+            {'name': 'linkedin', 'required': False,},
+            {'name': 'instagram', 'required': False,},
```

Comment:
Which tool is adding these trailing commas, assuming it's a tool and not you doing it manually? :stuck_out_tongue: I can't reproduce it with python-black.

Let's keep unnecessary changes to a minimum. These, in particular, don't seem very helpful. The commas at the very end of the lines are nice because they allow us to add, remove, and reorder items at the end without worrying about commas and without adding noise to the diff because an adjacent line had to have a trailing comma changed.  However, the trailing comma within the dictionary doesn't help with this.